komorebic/src/main.rs

```diff
@@ -154,6 +154,7 @@ gen_target_subcommand_args! {
     FocusWorkspace,
     FocusWorkspaces,
     MoveWorkspaceToMonitor,
+    SwapWorkspacesToMonitor,
```

Comment:
So this command will swap the workspaces of the currently focused monitor with the monitor given as `target`? In this case let's rename to `SwapWorkspacesWithMonitor`